clang/include/clang/Format/Format.h

```diff
@@ -225,6 +225,22 @@ struct FormatStyle {
     ///   bbb = 2;
     /// \endcode
     bool AlignCompound;
+    /// Only for ``AlignConsecutiveDeclarations``. Whether function pointers
+    /// are aligned.
```

Comment:
```suggestion
    /// Only for ``AlignConsecutiveDeclarations``. Whether function pointers are
    /// aligned.
```
Please regenerate the rst file after reflowing the comment.